.pipelines/azure-pipeline-build.yml

```diff
@@ -257,8 +257,10 @@ stages:
         displayName: "ESRP CodeSigning for Prometheus Golang Linux Reference App"
         inputs:
           ConnectedServiceName: "ESRPServiceConnectionPrometheus"  
-          AppRegistrationClientId: '73f8d5f9-b507-497f-b698-4ed00fcba5a3' 
-          AppRegistrationTenantId: '72f988bf-86f1-41af-91ab-2d7cd011db47' 
+          UseMSIAuthentication: true
+          AppRegistrationClientId: 'bf21d5fe-78bd-45d2-b04d-c61f2e090c9a' 
+          AppRegistrationTenantId: '33e01921-4d64-4f8c-a055-5bdaffd5e33d' 
+          EsrpClientId: "73f8d5f9-b507-497f-b698-4ed00fcba5a3"
```

Comment:
Hey @rashmichandrashekar Can you add a small doc in our InfraInsights.wiki of where the ESRP resource is and what access is needed + anything else that might be important for this so that its easier to figure this out if it needs to be updated?

Reply:
Thanks Kaveesh, added - https://msazure.visualstudio.com/InfrastructureInsights/_git/InfrastructureInsights.wiki?path=/ManagedPrometheus/Docs/ESRP.md&version=GBwikiMaster&_a=preview